Libplanet/Blockchain/BlockChain.cs

```diff
@@ -708,6 +708,11 @@ bool renderActions
                     nonceDeltas[txSigner] = nonceDelta + 1;
                 }
 
+                if (evaluateActions)
+                {
+                    ExecuteActions(block, renderActions);
```

Comment:
This causes inconsistency between states and a blockchain, since there would be a moment that states made from a block not appended yet are visible through `GetState()` method.

Reply:
`GetState()` will be okay for now because the `Tip` is not changed yet. But we need to address the inconsistency between states and a blockchain as you said.